FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItemGroup.kt

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+class CommandItemGroup {
```

Comment:
(may not be relevant if we go with a more static CCB design, but should groups also have an identifier property?)